### DISCUSSION TITLE
Export mesh.extract_feature_edges()

### DISCUSSION THREAD

**Discussion by stockerphilipp** [pyvista/pyvista#3604] [Q&A] [answered]
Using following code, I tried to export feature edges as an .stl file:
`features = mesh.extract_feature_edges(30)`
`features.save("file_path.ply")`

Unfortunately this results in an empty .stl file:
`Visualization Toolkit generated SLA File NUL NUL `...

Ploting the features works perfectly:
`p = pv.Plotter()`
`p.add_mesh(mesh, color=True)`
`p.add_mesh(features, color="red", line_width=5)`

Is there a way I can export my `features` as an .stl file?

**Comment by darikg** [comment, marked as answer]
I think STL can only store triangles, not individual edges

**Reply by banesullivan** [reply]
There are many other formats out there that do support edges, though, so please don't be discouraged. PyVista supports exporting to all of the formats handled by meshio: https://github.com/nschloe/meshio

[a comment by an automated account is not shown]
